README.md

```diff
@@ -2,29 +2,124 @@
 [![Codecov](https://codecov.io/gh/eclipse-iceoryx/iceoryx-rs/branch/master/graph/badge.svg?branch=master)](https://codecov.io/gh/eclipse-iceoryx/iceoryx-rs?branch=master)
 
 <p align="center">
 <img src="https://user-images.githubusercontent.com/8661268/114321508-64a6b000-9b1b-11eb-95ef-b84c91387cff.png" width="50%">
 </p>
 
-Experimental Rust bindings for the [iceoryx](https://github.com/eclipse-iceoryx/iceoryx) IPC middleware.
+Safe Rust bindings for [Eclipse iceoryx](https://github.com/eclipse-iceoryx/iceoryx).
 
-# clone and build
+1. [About](#about)
+2. [Examples](#examples)
+    - [How to start RouDi](#how-to-start-roudi)
+    - [Run the simple publisher and subscriber example](#run-the-simple-publisher-and-subscriber-example)
+3. [Limitations](#limitations)
 
-Clone the repository with
+## About
+
+Eclipse iceoryx is a true zero-copy, inter-process communication framework with the goal to boost
+autonomous driving with their demand on high data throughput and low latency. With its bandwidth and speed,
+iceoryx also fits well into other domains where low latency and transmitting large data structures
+is a concern. If you would like to know more about Eclipse iceoryx you can take a look at the
+`Getting started` section on [iceoryx.io](https://iceoryx.io) or the
+[README.md](https://github.com/eclipse-iceoryx/iceoryx/blob/master/README.md) of the main project.
+
+The Rust bindings are a work in progress and currently support only the pub-sub messaging pattern.
+Upcoming releases will close the gap and the goal is to have the Rust bindings as a first class citizen
+in the iceoryx ecosystem.
+
+This project started with the goal to create an awesome looking introspection TUI in Rust and led to
+[iceray](https://crates.io/crates/iceray). Check it out.
+
+## Examples
+
+Before you can run the example you have to build them first with
+```console
+cargo build --all --examples
 ```
-git clone https://github.com/elBoberido/iceoryx-rs.git
+
+In order to run an iceoryx application, the `RouDi` daemon needs to run.
+
+### How to start RouDi
+
+`RouDi` is the central daemon which takes care of resource management and connects the services
+when they register. After the registration phase it is not involved in the communication anymore.
+
+You can find more information about `RouDi` [here](https://iceoryx.io/v2.0.2/getting-started/overview/#roudi)
+and if you are more of a visual person, just scroll up a little bit on that page to view an amazing animation.
+
+In case you have iceoryx installed on your system, you can use the `iox-roudi` binary from that installation.
+If that's not the case, you can run `RouDi` with the following command from the root of your crate.
+
+```console
+find target -type f -wholename "*/iceoryx-install/bin/iox-roudi" -exec {} \;
 ```
 
-To build the examples run
+### Run the simple publisher and subscriber example
+
+A good introductory example to demonstrate the inter-process communication are `publisher_simple`
```

Comment:
I asked @FerdinandSpitzschnueffler for advise. The statement was: it sounds wrong and has to be completely rewritten. You speak of one (singular) good example and mix it with are (plural).

What about:
The executables `publisher_simple` and `subscriber_simple` are forming of a good introductory example.